### PR TITLE
use double quotes

### LINKED ISSUE #1001: The examples of using signing API with curl don't work (at least in Windows) because of single quotes
Both links with examples with `curl`: [in the blog post](https://blog.mozilla.org/addons/2015/11/20/signing-api-now-available/) and [in olympia docs](https://olympia.readthedocs.org/en/latest/topics/api/signing.html), use single quotes around `curl` parameters.

And it seems that at least Windows version of `curl` 7.43.0 doesn't work with single quotes, but works just fine with double. So I propose to correct this.

### PR DESCRIPTION
**Pull request by andymckay**
* consistently use " for curl commands which I think will work everywhere (I note that the curl docs also use this standard)
* should fix #1001

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -43,8 +43,8 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'olympia'
-copyright = u'2014, The Addons Crew'
+project = u'addons-server'
```
Comment (wagnerand) [inline docs/conf.py]: Nice!

File: docs/topics/api/signing.rst [hunk 2]
```diff
@@ -45,9 +45,9 @@ validation and you will be able to check its status.
 
     .. sourcecode:: bash
 
-        curl https://addons.mozilla.org/api/v3/addons/@my-addon/versions/1.0/
-            -g -XPUT --form 'upload=@build/my-addon.xpi'
-            -H 'Authorization: JWT <jwt-token>'
+        curl "https://addons.mozilla.org/api/v3/addons/@my-addon/versions/1.0/""
```
Comment (wagnerand) [inline docs/topics/api/signing.rst]: One too many?

**Comment by wagnerand** [comment]
r+wc

File: docs/topics/api/signing.rst (hunk 2 above)
Comment (andymckay) [inline docs/topics/api/signing.rst]: thanks

**Pull request merged**